README.md

```diff
@@ -1,57 +1,62 @@
-Visit [CurseForge](https://www.curseforge.com/wow/addons/azeroth-pilot-reloaded) for more information
-
-## Story 
-
-Here's why I am continuing Azeroth Auto Pilot's development;
-
-While using Azeroth Auto Pilot, I encountered various issues. At a point during the Warlords of Draenor campaign at level 35, I ended up coming to a hard stop due to a bug that completely broke the leveling process with the addon. I decided to go to AAP's Discord to get help, and I was told that AAP has gone unmaintained and its author/developer has ghosted their own support team. The addon has been left abandoned and, in some parts, broken, and so I was told, was way too messy for anybody to want to deal with. I was interested to see how messy it was, and believe me; it was messy!
-
-I don't blame anyone for not wanting to touch it, but I had 8 alts to level!
-
-Azeroth Auto Pilot's own support team suggested I take a look at the code and see what I could do with it.
-
-## Contribution
-If you wish to contribute, some features: 
-
-- Waypoint arrow
-- Quest Objectives
-- Asks if you want to do group quests
-- Auto flight paths
-- Auto talk to NPC's
-- Auto pick-up and hand-in
-- Auto skip cutscenes (Hold ctrl to not skip)
-- Shows button for quests with usable items (Keybindable)
-- UI allows moving and scaling
-- Chat commands
-- Faction specific questing
-- Multiple paths
-- Hidable windows
-
-## Settings 
-
-### Settings page
-Go through Interface > Addons > Azeroth Pilot Reloaded.
-
-You can use /apr as the prefix for chat commands.
-
-### Commandes
-Commandes list: 
-- `/apr skip` will skip the current step.
-- `/apr rollback` will rollback to the previous step if is not already completed
-- `/apr reset` will reset your pathing and recheck what you have completed. Useful if you accidentally /apr skipped too far or it doesn't do the job at fixing something.
-- `/apr hideriding` to hide the riding learning skill message
-- `/apr showriding` to show the riding learning skill message
-
-### Tips
-You can hold ctrl right before doing something that would otherwise be automated, and it will temporarily disable the skipping of that thing, such as a cutscene.
-
-### Useful Links
-[APR Curse Forge page](https://www.curseforge.com/wow/addons/azeroth-pilot-reloaded)
-
-[APR Discord](https://discord.com/invite/YgcdybKdWX)
-
-[AAP Discord](https://discord.gg/hpKyGnQ)
+![image](https://user-images.githubusercontent.com/28711160/205469925-6c73a0d6-0047-478a-aa12-bb9a5024c514.png)
+<h3 style="text-align: left;"><span style="font-size: 18px; font-family: arial;">The continuation of <em>Azeroth Auto Pilot</em>, a leveling AddOn.</span></h3>
+<p style="text-align: left;">&nbsp;</p>
+<hr />
+<p style="text-align: center;"><span style="font-size: 24px; font-family: arial;"><strong>If you wish to contribute, head to our Github </strong><strong><a href="https://github.com/Rycia/azeroth-pilot-reloaded" target="_blank" rel="noopener noreferrer">here.</a></strong></span></p>
```

Comment:
Line 5 needs to be changed to direct us to the Curseforge page, `'Visit [CurseForge](https://www.curseforge.com/wow/addons/azeroth-pilot-reloaded) for more information'` as before.
Change this line to:

`<p style="text-align: center;"><span style="font-size: 24px; font-family: arial;"><strong>Check out the Curseforge page </strong><strong><a href="https://www.curseforge.com/wow/addons/azeroth-pilot-reloaded" target="_blank" rel="noopener noreferrer">here.</a></strong></span></p>`